.insomnia/Request/req_8f843e785e1c442d961a9ed0437d289c.yml

```
@@ -0,0 +1,37 @@
+_id: req_8f843e785e1c442d961a9ed0437d289c
+type: Request
+parentId: fld_80bc7ee1b14547dc860545b5af4878c5
+modified: 1681910084632
+created: 1681907101878
+url: "{{ _.base_url }}/api/v1/reviews/{{ _.review_id }}/autoresolve"
+name: Auto Resolve
+description: >-
+  This endpoint triggerrs an `autoresolve` job for the given review
+
+
+
+  Params:
+
+  - `autoresolve_percentage` (int) value should be between `0` and `99`
```

The backend doesn't validate this value. I found that 100 is also acceptable and has a valid use-case. I suggest the below change:
```suggestion
  - `autoresolve_percentage` (int) value should be between `0` and `100`, inclusive.
```